### PR TITLE
Address PR feedback on #19520

### PR DESCRIPTION
**Pull request by DLehenbauer**
Fixes one package that was inadvertently changed to CJS.

Removes redundant inclusion of "tsconfig.base.json"

### CODE REVIEW

File: azure/packages/azure-client/src/test/tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
 	"extends": [
-		"../../../../../common/build/build-common/tsconfig.base.json",
```
Comment (DLehenbauer) [inline azure/packages/azure-client/src/test/tsconfig.json:3]: Fixed all pre-existing occurrences.

**Comment by jason-ha** [comment]
experimental/PropertyDDS/packages/property-shared-tree-interop/package.json doesn't look repaired. It became a double CJS build.

**Review by jason-ha — COMMENTED** [review]
Current changes look good - still needs repair to experimental/PropertyDDS/packages/property-shared-tree-interop

**Comment by DLehenbauer** [comment]
@jason-ha - Thanks, now includes 'property-shared-tree-interop'.

**Comment by jason-ha** [comment]
> property-shared-tree-interop

It is now doing double ESM build into dist folder, yeah?
It becomes easier to fix after #19596

**Review by jason-ha — CHANGES_REQUESTED** [review]
property-shared-tree-interop is now double building ESM into dist and nothing to lib.
(It looks like we should have a secondary validation step that validates we have generated CJS/ESM for dist/lib respectively.)

**Comment by DLehenbauer** [comment]
@jason-ha - I've removed 'property-shared-tree-interop' from this PR, per #19611.

**Review by jason-ha — APPROVED** [review]

**Review by Abe27342 — APPROVED** [review]

**Pull request merged**